pkg/products/amqonline/reconciler.go

```diff
@@ -350,3 +359,53 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcilePrometheusRule(ctx context.Context, installation *integreatlyv1alpha1.Installation, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-sli",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{
+		{
+			Alert: fmt.Sprintf("AMQ-SLI-1.1: AMQ Online console is not available in namespace %s", r.Config.GetNamespace()),
```

Comment:
@david-martin I think this was asked already, but probably on an earlier iteration of this PR: is there a naming convention for these "Alert" fields? For some reason most alerts seem to have a short string here (a couple of words camelCased into a single identifier usually), then something more descriptive in `Annotations.message`. I kinda like the idea of this top-level thing being immediately descriptive, but I don't have a strong opinion either way.

Reply:
@grdryn The template from Rob suggests we should probably switch it around to a more abstract alert and keep the more descriptive part for the message, I'll Refactor to adhere to this standard   https://github.com/EnMasseProject/enmasse/blob/master/controller-manager/src/main/resources/templates/PrometheusRules-enmasse.yaml

Reply:
Sounds good! :+1:

Reply:
@grdryn @HubertStefanski I have an issue (downstream/private https://issues.redhat.com/browse/INTLY-4624) for reviewing Alert names for consistency. Including the SLI or SLO is something to consider for that.
Given the distributed nature of where alerts are defined for RHMI, it needs some thought on how best to make updates to all alerts.
Going with the current defacto pattern makes sense for this PR.